troubleshooting/known_issues/iuf_error_argument_list_too_long.md

```diff
@@ -0,0 +1,91 @@
+# IUF Error: `exec /usr/local/bin/argoexec: argument list too long`
+
+Follow this procedure if an IUF step fails with the error: `exec /usr/local/bin/argoexec: argument list too long`.
+
+1. Get the name of the template that failed. For example, in the error below the `s3-upload` template failed.
+
+    ```bash
+    2024-03-11T20:40:54.348210Z INFO [cpe-23-12-3-s3-upload          ] BEG s3-upload(0)
+    2024-03-11T20:40:55.385227Z DBG  [cpe-23-12-3-s3-upload          ]       2024-03-11T20:40:54.569036586Z exec /usr/local/bin/argoexec: argument list too long
+    2024-03-11T20:41:03.980113Z ERR  [cpe-23-12-3-s3-upload          ] END s3-upload(0) [Failed]
+    ```
+
+1. This error is due to the script in that template being too long. This can be fixed by not echoing
+`{{inputs.parameters.global_params}}` in the template. Instead, write `{{inputs.parameters.global_params}}` to a file and then
+reference the file.
```

Comment:
```suggestion
1. Resolve the error by not echoing`{{inputs.parameters.global_params}}` in the template.
    Instead, write `{{inputs.parameters.global_params}}` to a file and then reference the file.
```

I would add the "This error is due to the script in that template being too long." line to the previous step and focus on the action for this step.